src/ploomber_core/warnings.py

```diff
@@ -6,4 +10,26 @@ class PloomberDeprecationWarning(FutureWarning):
         Changed superclass from UserWarning to FutureWarning
     """
 
-    pass
+    # @
+    def __init__(self, message):
+        print("message: ", message)
+        self.message = message
+
+
+def deprecation_warning(message, module_telemetry=None):
```

Comment:
let's make it mandatory

```suggestion
def deprecation_warning(telemetry, message):
```

Reply:
fix in [557b5ad](https://github.com/ploomber/core/pull/65/commits/557b5ad96968747255a5006d186873e84d2e37b8)